### DISCUSSION TITLE
MAC M1 GITHUB COPILOT not working in vscode

### DISCUSSION THREAD

**Discussion by vegetax** [community/community#19286] [Copilot Conversations] [answered]
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/38912494/175335794-751f336a-52da-4bde-8714-9ecc7b1d2c27.png">
all setting is well, but no suggest

**Comment by vegetax** [comment, marked as answer]
got the answer,  it's my network problem......

**Comment by damoski2** [comment]
The Solutions that worked for me and some other dev are:

1.Reload Your VSCode...
2.configuring SSH with your terminal also improves chance for it to work
3.Check your internet connection as copilot occasionally disconnects if transfer speed is low
4.Update VsCode
